src/services/reference-manager.ts

```diff
@@ -10,7 +10,7 @@
 
 import {logFactory} from '../platform/log-web.js';
 
-const log = logFactory('resource-mgr');
+const log = logFactory('reference-mgr');
 
```

Comment:
Can we extract this rename into a separate PR?

Reply:
Why?